session/auth/authprovider/tokenseed.go

```diff
@@ -53,9 +53,8 @@ func (ts *tokenSeeds) getSeed(host string) ([]byte, error) {
 			return nil, err
 		}
 	} else {
-		if err := json.Unmarshal(dt, &ts.m); err != nil {
-			return nil, errors.Wrapf(err, "failed to parse %s", fp)
-		}
+		// ignore error in case of crash during previous marshal
+		_ = json.Unmarshal(dt, &ts.m)
```

Comment:
On error, do we need to clear `ts.m` explicitly here?

Reply:
It's unlikely that something would be set there, but I guess if it is, there is no need to ignore it. If some parts were parsed then we can still use these seeds.